docs/cn/FAQ.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: global
+title: FAQ
+nickname: FAQ
+group: User Guide
+priority: 2
+---
+
+## Alluxio 常见问题
+
+1. 什么是Alluxio?
+
+   [Alluxio](http://www.alluxio.org/)（之前名为Tachyon），是一个开源的具有内存级速度的虚拟分布式存储系统，
+   使得应用程序可以以内存级速度与任何存储系统中的数据进行交互。
+
+2. Alluxio可以运行在哪些平台和Java版本上?
+
+   Alluxio可以在Linux / Mac的各种发行版上使用Java 1.7或更高版本运行。
+
+3. 是否有简单的方法来查看群集的状态和运行状况？
+  
+   是的，一旦安装好Alluxio，可以通过访问`http://master_ip:19999`的UI界面来查看。
+
+4. 如果向集群添加新节点，为了平衡节点之间的磁盘空间利用率，Alluxio是否会进行重新平衡（即将缓存的块移动到新节点）？
```

Comment:
I have fixed FAQ.md both the [English version](https://github.com/Alluxio/alluxio/pull/4848) and the Chinese version. Please check again, many thanks to you. @ifcharming